pyvista/trame/jupyter.py

```diff
@@ -130,6 +130,8 @@ def launch_server(server=None, port=None, host=None):
     if not pyvista.global_theme.trame.enable_vtk_warnings:
         vtk_logger = logging.getLogger("vtkmodules.web.render_window_serializer")
         vtk_logger.disabled = True
+        trame_vtk_logger = logging.getLogger("trame_vtk.modules.vtk.serializers.serialize")
+        trame_vtk_logger.disabled = True
```

Comment:
There are actually several loggers in the `trame_vtk.modules.vtk.serializers` directory now. The parent one in the `__init__.py` file is set to `CRITICAL` level by default (and there are currently no critical messages, so nothing is printed by default), unless the `TRAME_SERIALIZE_DEBUG` environment variable is set, in which case all messages will be printed.

I wonder if disabling the parent `trame_vtk.modules.vtk.serializers` will disable all of the children too?

Reply:
> I wonder if disabling the parent trame_vtk.modules.vtk.serializers will disable all of the children too?

I had initially tried this without luck... just tried again and it works now... something must have change in the last few commits?

Reply:
Yes, I stopped setting the log level on the children, which means that they default to the parent log level (which makes it easier for us to turn them all on or off). And the parent now defaults to `CRITICAL`, so messages are no longer printed by default.

Reply:
Right, so this isn't even needed, thanks!